console/src/components/Services/Actions/Permissions/Permissions.js

```diff
@@ -119,7 +120,7 @@ const Permissions = ({
             permType: queryType,
             className,
             editIcon: editIcon,
-            onClick,
+            onClick: !readOnlyMode && onClick,
```

Comment:
It results in an error:

<img width="1545" alt="Screenshot 2020-06-03 at 16 31 17" src="https://user-images.githubusercontent.com/9019397/83649606-ad1a4680-a5b7-11ea-93b2-6080ecf2a485.png">